code/common/platform.h

```diff
@@ -4,11 +4,18 @@ namespace common
 {
   enum class platform
   {
-    windows
+    windows,
+    linuxplatform // plain 'linux' is a g++ predefined macro
```

Comment:
`linux_x11` might be better here for now.

Reply:
Addressed in 02966ccf343cdd5a0222c38d92fd379eaaf40d6d